src/components/TeacherComponents/OnboardingForm/OnboardingForm.tsx

```diff
@@ -91,6 +104,15 @@ const OnboardingForm = ({
           );
         }
       }
+
+      // Return the user to the page they originally arrived from
+      // or to the home page as a fallback
+      router.push(
+        toSafeRedirect(
```

Comment:
Sanitises the `returnTo` param from phishing attempts. Clerk should be doing this internally for `redirect_url` but since we're doing this redirect ourselves it's out responsibility.